src/components/QueryCondition.vue

```diff
@@ -0,0 +1,132 @@
+			:placeholder="$i18n('query-builder-input-value-placeholder')"
+			:disabled="selectedPropertyValueRelation === propertyValueRelation.Regardless"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapState } from 'vuex';
+import { TextInput } from '@wmde/wikit-vue-components';
+
+import PropertyLookup from '@/components/PropertyLookup.vue';
+import ValueTypeDropDown from '@/components/ValueTypeDropDown.vue';
+import SearchResult from '@/data-access/SearchResult';
+import PropertyValueRelation from '@/data-model/PropertyValueRelation';
+import Error from '@/data-model/Error';
+import allowedDatatypes from '@/allowedDataTypes';
+import Validator from '@/form/Validator';
+
+export default Vue.extend( {
+	name: 'QueryCondition',
+	data() {
+		return {
+			fieldErrors: {
+				property: null as null | Error,
+				value: null as null | Error,
+			},
+			limitedSupport: false,
+			propertyValueRelation: PropertyValueRelation,
+		};
+	},
+	methods: {
+		validate(): void {
+			const formValues = {
+				property: this.selectedProperty,
+				value: this.textInputValue,
+				propertyValueRelation: this.selectedPropertyValueRelation,
+			};
+			const validator = new Validator( formValues );
+			const validationResult = validator.validate();
+			this.fieldErrors = validationResult.fieldErrors;
+			if ( this.selectedProperty !== null ) {
+				this.validateForLimitedSupport( this.selectedProperty );
+			}
+		},
+		validateForLimitedSupport( selectedProperty: SearchResult ): void {
+			this.limitedSupport = false;
+			this.fieldErrors.property = null;
+			if ( selectedProperty && !allowedDatatypes.includes( selectedProperty.datatype ) ) {
+				this.selectedPropertyValueRelation = PropertyValueRelation.Regardless;
+				this.limitedSupport = true;
+				this.fieldErrors.property = {
+					type: 'warning',
+					message: 'query-builder-property-lookup-limited-support-note',
+				};
+			}
+		},
+	},
+	computed: {
+		selectedProperty: {
+			get(): SearchResult | null {
+				return this.$store.getters.property;
+			},
+			set( selectedProperty: SearchResult ): void {
+				this.selectedPropertyValueRelation = PropertyValueRelation.Matching;
+				this.validateForLimitedSupport( selectedProperty );
+				this.$store.dispatch( 'updateProperty', selectedProperty );
+			},
+		},
+		selectedPropertyValueRelation: {
+			get(): PropertyValueRelation {
+				return this.$store.getters.propertyValueRelation;
+			},
+			set( selectedPropertyValueRelation: PropertyValueRelation ): void {
+				if ( selectedPropertyValueRelation === PropertyValueRelation.Regardless ) {
+					this.textInputValue = '';
+				}
+				this.$store.dispatch( 'updatePropertyValueRelation', selectedPropertyValueRelation );
+			},
+		},
+		textInputValue: {
+			get(): string { return this.$store.getters.value; },
+			set( value: string ): void { this.$store.dispatch( 'updateValue', value ); },
+		},
+		...mapState( {
```

Comment:
We're not using this at all in this component

Reply:
Done